<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>18</y>
      <width>523</width>
      <height>523</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>523</width>
      <height>523</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>523</width>
      <height>523</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="selector">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Space</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Starting Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>End Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wall</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Pathfinder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Grid</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selector_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dijkstra's</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setSpaceType()</slot>
 </slots>
</ui>
